idd/Energy+.idd.in

```diff
@@ -50272,6 +50272,236 @@ ZoneHVAC:EquipmentConnections,
+       \object-list ZoneNames
+   A3, \field Zone Equipment Object Type
+       \required-field
+       \type choice
+       \key ZoneHVAC:TerminalUnit:VariableRefrigerantFlow
+       \key ZoneHVAC:AirDistributionUnit
+       \key ZoneHVAC:EnergyRecoveryVentilator
+       \key ZoneHVAC:EvaporativeCoolerUnit
+       \key ZoneHVAC:HybridUnitaryHVAC
+       \key ZoneHVAC:ForcedAir:UserDefined
+       \key ZoneHVAC:FourPipeFanCoil
+       \key ZoneHVAC:OutdoorAirUnit
+       \key ZoneHVAC:PackagedTerminalAirConditioner
+       \key ZoneHVAC:PackagedTerminalHeatPump
+       \key ZoneHVAC:UnitHeater
+       \key ZoneHVAC:UnitVentilator
+       \key ZoneHVAC:VentilatedSlab
+       \key ZoneHVAC:WaterToAirHeatPump
+       \key ZoneHVAC:WindowAirConditioner
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+       \key ZoneHVAC:Dehumidifier:DX
+       \key ZoneHVAC:IdealLoadsAirSystem
+       \key ZoneHVAC:RefrigerationChillerSet
+       \key WaterHeater:HeatPump:PumpedCondenser
+       \key WaterHeater:HeatPump:WrappedCondenser
+       \key HeatExchanger:AirToAir:FlatPlate
+       \key AirLoopHVAC:UnitarySystem
+   A4, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
+   A5, \field Zone Equipment Outlet Node Name
+       \note Only used for airflow equipment.
+       \note The outlet node from the zone equipment that will be split to the spaces.
+       \type node
+   A6, \field Thermostat Control Method
+       \note SingleSpace satisfies the thermostat in the Control Space Name
+       \note Maximum satisfies the thermostat in the connected spaces with the highest deviation from setpoint
+       \note Ideal ignores the Space Output Fractions and distributes the output to match the current space sensible loads.
+       \type choice
+       \key SingleSpace
+       \key Maximum
+       \key Ideal
+       \default SingleSpace
+   A7, \field Control Space Name
+       \note Used with SingleSpace thermostat control method
+       \type object-list
+       \object-list SpaceNames
+   A8, \field Space Fraction Method
+       \note The basis used to autosize the space output fractions.
+       \type choice
+       \default DesignCoolingLoad
+       \key DesignCoolingLoad
+       \key DesignHeatingLoad
+       \key FloorArea
+       \key Volume
+       \key PerimeterLength
+   A9, \field Space 1 Name
+       \begin-extensible
+       \required-field
+       \type object-list
+       \object-list SpaceNames
```

Comment:
Does this object need a min-fields? I'm trying to figure out where I can stop providing inputs for this object.

Reply:
Yes, minfilelds up to last field for space 1